### DISCUSSION TITLE
Make the board smaller to fit in a quarter-pane?

### DISCUSSION THREAD

**Discussion by jlharri** [roovo/obsidian-card-board#80] [Q&A] [answered]
Hello, 
I've started using Obsidian recently and am trying out Cardboard. I have a workspace configured with Cardboard displayed in a pane that's 1/4 the size of my screen. As you can see, the columns and text are large...I can only see 2 cards max in 2 columns. Are there any CSS snippets I can use to reduce the size of the text and columns to increase the amount of content displayed? Perhaps doubling it to display 4 columns and 2 rows of cards? 

Thank you, 
Jerry

**Comment by roovo** [comment, marked as answer]
Hiya - I have just been working on the plugin and have updated the css in the latest release to allow this.  See https://github.com/roovo/obsidian-card-board#scaling-board-size for info on how to do this.  Hopefully it is (or perhaps was) what you were after.

**Reply by roovo** [reply]
I've marked this as the answer - do feel free to comment (or un-mark) if you don't agree!!
